tools/accuracy_checker/accuracy_checker/annotation_converters/cmu_panoptic.py

```diff
@@ -0,0 +1,169 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import numpy as np
+import os
+
+import cv2
+
+from ..config import BoolField, PathField
+from ..utils import read_json, convert_bboxes_xywh_to_x1y1x2y2, check_file_existence
+from ..representation import PoseEstimation3dAnnotation
+from .format_converter import FileBasedAnnotationConverter, ConverterReturn
+
+
+def get_image_annotation(image_id, annotations_):
+    return list(filter(lambda x: x['image_id'] == image_id, annotations_))
+
+
+def get_label_map(full_annotation, use_full_label_map=False, has_background=False):
+    labels = full_annotation['categories']
+
+    if not use_full_label_map:
+        label_offset = 1 if has_background else 0
+        label_id_to_label = {label['id']: label_id + label_offset for label_id, label in enumerate(labels)}
+        label_map = {label_id + label_offset: label['name'] for label_id, label in enumerate(labels)}
+    else:
+        label_id_to_label = {label['id']: label['id'] for label in labels}
+        label_map = {label['id']: label['name'] for label in labels}
+
+    return label_map, label_id_to_label
+
+
+class CmuPanopticKeypointsConverter(FileBasedAnnotationConverter):
+    __provider__ = 'cmu_panoptic_keypoints'
+    annotation_types = (PoseEstimation3dAnnotation,)
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update(
+            {
+                'images_dir': PathField(
+                    is_directory=True, optional=True,
+                    description='path to dataset images, used only for content existence check'
+                )
+            }
+        )
+        return parameters
+
+    def configure(self):
+        super().configure()
+        self.images_dir = self.get_value_from_config('images_dir') or self.annotation_file.parent
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        data = {}
+        images_dir_name = 'hdImgs'
+        labels_dir_name = 'hdPose3d_stage1_coco19'
+        label_prefix = 'body3DScene'
+        dataset_dir = os.path.join(str(self.images_dir), 'data')
```

Comment:
We prefer to use [pathlib](https://docs.python.org/3/library/pathlib.html) instead `os.path`